fil_actors_shared/src/actor_versions.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use serde::{Deserialize, Serialize};
+
+/// Enum representing supported actor versions.
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    PartialEq,
+    Eq,
+    Hash,
+    PartialOrd,
+    Ord,
+    Serialize,
+    Deserialize,
+    strum::Display,
+    strum::EnumString,
+    strum::FromRepr,
+)]
+#[repr(u8)]
+pub enum ActorVersion {
+    #[strum(to_string = "v8")]
+    V8 = 8,
+
+    #[strum(to_string = "v9")]
+    V9 = 9,
```

Comment:
I _think_ you don't need to set the number for subsequent variants. That is, `v9` should automatically go to `9`.